### PR TITLE
🌱 Remove test suffix from pivoting function file

### PR DESCRIPTION
**Pull request by mboukhalfa**
https://github.com/metal3-io/cluster-api-provider-metal3/pull/728 follow up check comment https://github.com/metal3-io/cluster-api-provider-metal3/pull/728#discussion_r1030632080

### CODE REVIEW

File: test/e2e/pivoting.go [hunk 1]
```diff
@@ -262,7 +262,7 @@ type RemoveIronicInput struct {
 
 func removeIronic(ctx context.Context, inputGetter func() RemoveIronicInput) {
 	input := inputGetter()
-	if input.IsDeployment == true {
+	if input.IsDeployment {
```
Comment (mboukhalfa) [inline test/e2e/pivoting.go:265]: golangci-lint: 
```
 test/e2e/pivoting.go:265:5: S1002: should omit comparison to bool constant, can be simplified to `input.IsDeployment` (gosimple)
	if input.IsDeployment == true { 
```

Comment (lentzi90) [reply test/e2e/pivoting.go:265]: Indeed, this is one of the benefits of using booleans directly instead of strings. :+1:

**Review by lentzi90 — COMMENTED** [review]
/lgtm

**Review by furkatgofurov7 — APPROVED** [review]

**Pull request merged**